modules/core/shared/src/main/scala-3/io/circe/Derivation.scala

```diff
@@ -133,37 +130,62 @@ private[circe] trait DerivedDecoder[A] extends DerivedInstance[A] with Decoder[A
 }
 
 private[circe] trait EncoderDerivation:
-  inline final def derived[A](using inline A: Mirror.Of[A]): Encoder.AsObject[A] =
+  inline final def derived[A](using inline A: LazyMirror[A], inline E: Encoders[A]): Encoder.AsObject[A] =
```

Comment:
I was kinda surprised this wasn't reported as binary incompatible. Do `inline` params not matter for bin compat?

Reply:
We might have to check for tasty-compatiblity here 
https://github.com/scalacenter/tasty-mima

it is possible that that is a missing check in mima, @SethTisue do you know?